EventFilter/EcalRawToDigi/src/MatacqRawEvent.cc

```diff
@@ -137,7 +136,7 @@ void MatacqRawEvent::setRawData(const unsigned char* pData, size_t maxSize){
   const int nCh = getChannelCount();
   channelData.resize(nCh);
 
-  pData16 = (int16le_t*) (begin+headerLen); 
+  pData16 = (int16le_t*) (pData+headerLen); 
```

Comment:
Done, but deleting line 91 and not using "daqHeaderLen". doesn't that make line 86  and "daqHeaderLen" useless as well? Or does this length variable has another use i'm not aware of?